### ISSUE TITLE
Colour contrast for some elements is not sufficient for WCAG guidelines

### ISSUE THREAD

**Issue by marxjohnson** [Laboratoire-de-Chemoinformatique/moodle-tiny_molstructure#15]
Out-of-the-box, ChemDoodle sets each element to a different colour. The trouble is, some of the colours don't have sufficient contrast to be readable against the white canvas. For example, if you select Helium (He), it is very bright blue and you cannot read it. For compliance with WCAG 2.0 AA, text contrast should be at least 4.5:1 (you can check it here: https://webaim.org/resources/contrastchecker/).

According to [web.chemdoodle.com](web.chemdoodle.com),

> The ChemDoodle Web Components library helps to make any product ADA (Americans with Disabilities Act) compliant as it is fully customizable. Add in sounds to sketcher actions using our API, provide color themes, control keyboard input.

It is possible for the Tiny plugin to provide a custom colour scheme when loading ChemDoodle so that the colours it uses are WCAG-compliant (maybe even just use black-on-white for element symbols, unless the colours have particular affordance for those who can see them?)
